tests/simpletest.py

```diff
@@ -302,6 +302,64 @@ def test_create_with_hash_seeds_and_compare(self):
         self.assertEqual(bf1_seeds, bf2_seeds)
         self.assertEqual(bf1_base64, bf2_base64)
 
+    def test_bit_array(self):
+        bf = pybloomfilter.BloomFilter(1000, 0.01, self.tempfile.name)
```

Comment:
Does this need to be closed explicitly as well?

Suggest to add `bf.close()` in the end so to not leave discriptor open. Although not entirely sure if it’s required.

Reply:
I think we may in fact not need it at all as it's handled during garbage collection (\_\_dealloc\_\_), so I'm removing it